@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -479,173 +479,153 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
     ): void {
         this.processFeatureCommon(env);
 
-        const techniqueIndex = techniques[0]._index;
-        if (techniqueIndex === undefined) {
-            throw new Error(
-                "OmvDecodedTileEmitter#processPolygonFeature: Internal error - No technique index"
-            );
-        }
-
-        const meshBuffers = this.findOrCreateMeshBuffers(techniqueIndex, GeometryType.Polygon);
-        if (meshBuffers === undefined) {
-            return;
-        }
-        const { positions, textureCoordinates, colors, indices, edgeIndices, groups } = meshBuffers;
+        techniques.forEach(technique => {
+            if (technique === undefined) {
+                return;
+            }
```

Comment:
Please minimize patch and nesting.

The whole interior of foreach can be actually new method called with `technique` ...
```
techniques.forEach(technique => {
     this.processPolygonFeatureInt(layer, geometry, ...., technique)
})
```
(makes patch and review smaller and a little less nesting is needed - only gains)